src/EmailReplyParser/Parser/EmailParser.php

```diff
@@ -21,7 +21,7 @@ class EmailParser
     /**
      * Regex to match signatures
      */
-    const SIG_REGEX   = '/(?:^\s*--|^\s*__|^-\w|^-- $)|(?:^Sent from my (?:\s*\w+){1,3})$/s';
+    const SIG_REGEX   = '/(?:^\s*--|^\s*__|^-\w|^-- $)|(?:^(Sent from my)|(发自我的) (?:\s*\w+){1,3})|(?:^나의(?:\s*\w*)(에서\s보냄)|(から送信))$/s';
```

Comment:
these should be non-capturing groups.
And putting the ``|`` outside the new groups looks wrong to me, as it changes the regex entirely for the English case (the new English regex is anchored at the beginning, but not at the end)